play/src/pusher/enums/EnvironmentVariable.ts

```diff
@@ -126,6 +127,7 @@ export const OPID_SCOPE = env.OPID_SCOPE || "openid email";
 export const OPID_PROMPT = env.OPID_PROMPT || "login";
 export const OPID_USERNAME_CLAIM = env.OPID_USERNAME_CLAIM || "username";
 export const OPID_LOCALE_CLAIM = env.OPID_LOCALE_CLAIM || "locale";
+export const USERNAME_POLICY = env.USERNAME_POLICY || "USER_INPUT";
```

Comment:
We should modify the contrib/docker/:

Add the variable in docker-compose.prod.yaml and in .env.template